arelle/plugin/validate/ESEF/ESEF_Current/ValidateXbrlFinally.py

```diff
@@ -147,19 +147,6 @@ def validateXbrlFinally(val: ValidateXbrl, *args: Any, **kwargs: Any) -> None:
     modelXbrl.profileActivity("... filer DTS checks", minTimeToShow=1.0)
 
     if getDisclosureSystemYear(modelXbrl) >= 2023:
-        instanceNumber = 0
-        if modelXbrl.fileSource.dir:
-            for file in modelXbrl.fileSource.dir:
-                if not file.endswith("/"):
-                    from arelle.ModelDocument import Type
-                    fileType = Type.identify(modelXbrl.fileSource, "{}/{}".format(modelXbrl.fileSource.basefile, file))
-                    if fileType == Type.INLINEXBRL:
-                        instanceNumber += 1
-            if instanceNumber > 1:
-                modelXbrl.error("ESEF.4.1.1.SingleXhtmlFiles",
```

Comment:
Test cases for `2.5.4` and `2.6.2` both explicitly check multi xhtml report packages.